04_flora_cnell/src/data_utils.R

```diff
@@ -0,0 +1,38 @@
+get_npn_layer <- function(layer_name, npn_layers, layer_date){
+  
+  layer_id <- npn_layers %>% 
+    filter(title == layer_name) %>%
+    pull(name)
+
+  if(!is.null(layer_date)){
+    label_date <- 'avg'
+  } else{
+    label_date <- layer_date
+  }
+  file_out <- gsub(':', '_', sprintf('out/%s_%s.tiff', layer_id, label_date))
+  npn_download_geospatial(coverage_id = layer_id,
+                          date = layer_date,
+                          format = 'geotiff',
+                          output_path = file_out
+  )
+  
+  return(file_out)
+  
+}
+prep_plot_df <- function(current_timing, proj){
```

Comment:
Since you use this function to process 3 rasters, I'd suggest making the variable names (e..g 'current_timing', 'anomolay_rast') more generic. But I think it would be good to rename this function to better reflect what it is achieving, e.g. `munge_npn_data()`